Yehor/reactNetflix/src/interfaces/IMovies.ts

```diff
@@ -0,0 +1,7 @@
+import IMovie from "./IMovie";
+
+interface IMovies {
+  movies: IMovie[];
```

Comment:
Remove.
Usually we don't create separate interfaces for arrays. Just use IMovie[] where it's needed.